08.ls_object/ls.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative 'printer'
+
+class LS
+  def initialize(options)
+    @directory = options[:directory]
+    @long_format = options[:long_format]
+    @files = get_files(options[:hidden_files], options[:reverse])
+  end
+
+  def output
+    printer = Printer.new(@directory, @long_format, @files)
```

Comment:
```suggestion
    printer = Printer.new(@file_details, @long_format)
```

のようにしてこの時点でFileDetailの配列を渡してしまった方がシンプルになりそうです